README.rst

```diff
@@ -55,6 +55,7 @@ For general language neutral documentation of vSphere Management API see:
 
 Python Support
 ==============
+* pyVmomi 6.7.3 supports 2.7, 3.3, 3.4, 3.5, 3.6 & 3.7
```

Comment:
I think we can remove the old releases and just say
pyVmomi supports 2.7 and 3.3+
For users who are using old releases, they can always refer to the corresponding version README.

Reply:
Maybe 2.7.x and 3.3+

Reply:
Done